### ISSUE TITLE
.gitignore

### ISSUE THREAD

**Issue by soncco** [ivandoric/d8theming#2]
Is there a .gitignore for node_modules folder or other files?

**Comment by ivandoric** [comment]
No, but you can make one yourself. Definitely ignore node_modules, always.
